### PR TITLE
More copy editing for invalidation.md

### PR DESCRIPTION
**Pull request by chrishtr**

### CODE REVIEW

File: book/invalidation.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ prev: embeds
 next: skipped
 ...
 
-Compositing makes animations smoother, but
+Compositing (See Chapter 13) makes animations smoother, but
```
Comment (pavpanchekha) [inline book/invalidation.md:8]: Do we really want this? also shouldn't "See" be lower case?

File: book/invalidation.md [hunk 2]
```diff
@@ -758,8 +758,7 @@ very specific sequence of changes.
 [under-invalidation]: https://developer.chrome.com/articles/layoutng/#under-invalidation
 [hard-to-find]: https://developer.chrome.com/articles/layoutng/#correctness
 
-
-Recursive invalidation
+Recursive Invalidation
```
Comment (pavpanchekha) [inline book/invalidation.md:761]: I'm not doing this in my PRs (I'll fix it in post, as in, when I copy over changes to the copyeditor's version) but it's worth doing so let's keep it in.

File: book/invalidation.md [hunk 3]
```diff
@@ -2077,7 +2066,7 @@ everything! I won't implement it, but paint can be made a lot faster
 too---see Exercise 16-10.)
 
 [^other-phases]:  It might also be pretty laggy on large pages due to the
-composite-raster-draw cycle being fairly slow, depending on which exercises you
+composite--raster--draw cycle being fairly slow, depending on which exercises you
```
Comment (pavpanchekha) [inline book/invalidation.md:2069]: Uhh, does this work right on web? I've been inserting unicode en dashes, but if it works it works.

**Review by pavpanchekha — APPROVED** [review]

File: book/invalidation.md (hunk 1 above)
Comment (chrishtr) [reply book/invalidation.md:8]: appears to work

File: book/invalidation.md (hunk 3 above)
Comment (chrishtr) [reply book/invalidation.md:2069]: appears to work

File: book/invalidation.md (hunk 1 above)
Comment (chrishtr) [inline book/invalidation.md:8]:
```suggestion
Compositing (see Chapter 13) makes animations smoother, but
```

**Pull request merged**